modules/apps/object/object-storage-salesforce-test/src/testIntegration/java/com/liferay/object/storage/salesforce/internal/rest/manager/v1_0/test/SalesforceObjectEntryManagerImplTest.java

```diff
@@ -214,6 +215,20 @@ public void setUp() throws Exception {
 				_objectDefinition.getObjectDefinitionId()
 			).build());
 
+		ObjectFieldUtil.addCustomObjectField(
+			new BooleanObjectFieldBuilder(
+			).externalReferenceCode(
+				"Flagged__c"
+			).userId(
+				adminUser.getUserId()
+			).labelMap(
+				LocalizedMapUtil.getLocalizedMap("Flagged")
+			).name(
+				"flagged"
+			).objectDefinitionId(
+				_objectDefinition.getObjectDefinitionId()
+			).build());
```

Comment:
Please sort this as well, it should come after [this](https://github.com/liferay-objects/liferay-portal/pull/3174/files#diff-82f88640eff05dedc724f92e76be2cf21cd6e4ffae22f9cba25e62e361ced5d0R160-R172)